docs/getting-started-guides/ibm-softlayer.md

```diff
@@ -0,0 +1,47 @@
+---
```

Comment:
Thanks for making a new page! You'll also need to add it to the left navigation. Please find a spot in /_data/guides.yml